assets/cubyz/biomes/decorative/stone_pit.zig.zon

```diff
@@ -0,0 +1,50 @@
+.{
+	.properties = .{
+	},
+	.radius = 10,
+	.chance = 0
+	.minHeight = 10,
+	.maxHeight = 20,
+	.roughness = 1,
+	.mountains = 30,
+	.hills = 10,
+	.stoneBlock = "cubyz:stone",
+	.parentBiomes = .{
+		.{
+			.id = "cubyz:grassland",
+			.chance = 5,
+		},
+		.{
+			.id = "cubyz:autumn/dead_forest",
+			.chance = 12,
+		},
+		.{
+			.id = "cubyz:autumn/mixed_forest",
+			.chance = 5,
+		},
+		.{
+			.id = "cubyz:autumn/forest",
+			.chance = 7,
+		},
+		.{
+			.id = "cubyz:rocky_grassland",
+			.chance = 32,
+		},
+		.{
+			.id = "cubyz:taiga",
+			.chance = 6,
+		},
+		.{
+			.id = "cubyz:forest/base",
+			.chance = 4,
+		},
+		.{
+			.id = "cubyz:tundra/base",
+			.chance = 8,
+		},
+		.{
+			.id = "cubyz:tundra/patchy",
+			.chance = 16,
+		},
+	},
+}
```

Comment:
Stone pits don't appear to work as intended:
![Screenshot at 2025-03-01 11-05-17](https://github.com/user-attachments/assets/4c3bb87a-87b5-4de3-975e-5c780bbfee5b)